README.md

```diff
@@ -27,255 +20,169 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
+| java           | >=11, <20                     | All          |
+| docker         | ^20.10.8                      | All          |
+| docker-compose | ^1.29.0                       | All          |
+| Windows        | 10 (OS build 20262 or higher) | Only Windows |
+| WSL            | 2                             | Only Windows |
 
-Since the container is running detached, you will not see any logs. However, you can follow the logs by running:
+You can check if these packages are installed and functioning by running:
 
 ```bash
-docker-compose logs -f
+git version
+java -version
+docker version
+docker-compose version
+docker run hello-world
 ```
 
-Use CTRL+C to stop monitoring the logs and return to a prompt.
-
-### Shutdown
-
-The deployment can be brought down by running:
-
-```bash
-docker-compose down
-```
+:::note
 
-### Manage example data
+Internally, the Java build process will use [Gradle Auto Provisioning](https://docs.gradle.org/current/userguide/toolchains.html#sec:provisioning)
+to download and use the appropriate Java version for building and testing.
 
-[Deephaven's examples repository](https://github.com/deephaven/examples) contains data sets that are useful when learning 
-to use Deephaven. These data sets are used extensively in Deephaven's documentation and are needed to run some examples. [Deephaven's examples repository](https://github.com/deephaven/examples) contains documentation on the available data sets and how to manage them. 
+On Windows, all commands must be run inside a WSL 2 terminal.
 
-If you have chosen a deployment with example data, the example data sets will be downloaded. Production deployments containing your own data will not need the example data sets.
+:::
 
+### Authentication
 
-To upgrade a deployment to the latest example data, run:
+Deephaven, by default, uses pre-shared key authentication to authenticate against unauthorized access. For more information on supported authentication methods:
 
-```bash
-docker-compose run examples download
-```
+- [How to configure and use pre-shared key authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-psk/)
+- [How to configure and use anonymous authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-anon/)
+- [How to configure and use username/password authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-uname-pw/)
+- [How to configure and use mTLS authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-mtls/)
+- [How to configure and use Keycload/OpenID authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-keycloak/)
 
-To see what other example data management commands are available, run:
+If using a randomly generated pre-shared key, you can find the key in the server logs:
 
 ```bash
-docker-compose run examples
+docker compose logs -f | grep "access through pre-shared key"
```

Comment:
Does this work with pip installed python?  Does this work if building from source?  I think the answer is no, so we've lost users.